index.bs

```diff
@@ -488,7 +491,7 @@ partial interface Window {
 
 The {{Window/getScreenDetails()}} method completes asynchronously, queuing work on the <dfn>window placement task source</dfn>.
 
-Instances of {{Window}} are created with an [internal slot](https://tc39.github.io/ecma262/#sec-object-internal-methods-and-internal-slots) named <dfn attribute for="Window">\[[screenDetails]]</dfn>, which is created with an initial value of `undefined`.
+Instances of {{Window}} are created with an [=internal slot=] named <dfn attribute for="Window">\[[screenDetails]]</dfn>, which is created with an initial value of `undefined`.
```

Comment:
Nit: if possible, use `[=/internal slot=]` (add the `/`) to make reference explicit.

(for all new links)

Reply:
I'll do that. FWIW, I note that the spec already asks Bikeshed to consider that the `/` is there when it's omitted through the [Assume Explicit For](https://tabatkins.github.io/bikeshed/#metadata-assume-explicit-for) parameter.